colibre/registration.py

```diff
@@ -762,6 +765,15 @@ def register_stellar_birth_densities(self, catalogue):
     return
 
 
+def register_los_star_veldisp(self, catalogue):
+    for aperture_size in [10, 30]:
+        veldisp = getattr(catalogue.apertures, f"veldisp_star_{aperture_size}_kpc")
+        los_veldisp = veldisp / np.sqrt(3.0)
```

Comment:
Let's add a name to this new field; otherwise, in the figures, it might be unclear what is plotted. See, e.g., x axis in your example

![xaxis](https://user-images.githubusercontent.com/20153933/135714277-b640c8bf-e169-4c07-a20f-ef15794a8ef2.png)


```suggestion
        los_veldisp = veldisp / np.sqrt(3.0)
        los_veldisp.name = f"LOS stellar velocity dispersion ({aperture_size} kpc)"
```